### PR TITLE
Fix write, gc rule issue

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- upgrade @ainize-team/ainize-js package version to `1.1.1`.
- remove container url from configuration data.
- fix write, GC rule issue. (currently, GC rules apply only to `message`)
- handle `FAIL` status and timeout for ainize request.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
   "dependencies": {
     "@ainblockchain/ain-js": "^1.6.0",
     "@ainblockchain/ain-util": "^1.1.9",
-    "@ainize-team/ainize-js": "1.0.4",
+    "@ainize-team/ainize-js": "^1.1.1",
```
Comment (platfowner) [inline package.json:44]: 🙌

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Review by minsulee2 — APPROVED** [review]
LGTM!

**Comment by jiyoung-an** [comment]
Thank you for quick review! @platfowner @minsulee2 now merging...

**Pull request merged**